app/assets/javascripts/admin/addon/templates/dashboard_general.hbs

```diff
@@ -12,12 +12,19 @@
               {{i18n "admin.dashboard.community_health"}}
             </a>
           </h2>
-          <PeriodChooser
-            @period={{this.period}}
-            @action={{action "changePeriod"}}
-            @content={{this.availablePeriods}}
-            @fullDay={{false}}
-          />
+          <span>
+            <PeriodChooser
+              @period={{this.period}}
+              @action={{action "changePeriod"}}
+              @content={{this.availablePeriods}}
+              @fullDay={{false}}
+            />
+            <DButton
```

Comment:
Might be good to add a `title` attribute here so people can find out what the button does (and also for accessibility)